format/Schema.fbs

```diff
@@ -20,7 +20,8 @@
 /// Format Version History.
 /// Version 1.0 - Forward and backwards compatibility guaranteed.
 /// Version 1.1 - Add Decimal256 (No format release).
-/// Version 1.2 (Pending)- Add Interval MONTH_DAY_NANO
+/// Version 1.2 - Add Interval MONTH_DAY_NANO (No format release).
```

Comment:
I don't know if we want to keep those "No format release" comments. They were probably related to the idea of doing format releases separate from implementation releases, but it doesn't seem like we're planning to ever do that?

@lidavidm Thoughts?

Reply:
I agree the comments are probably redundant. Implementations should just specify which format version they support.

Reply:
okay, i'll remove those `(No format release)` comments.